src/AccountProfile.jsx

```diff
@@ -1,9 +1,36 @@
 const accountId = props.accountId || context.accountId;
-const profile = props.profile || Social.get(`${accountId}/profile/**`, "final");
```

Comment:
If we're going to remove the fallback `Social.get(`${accountId}/profile/**`, "final")` logic, we'll need to make sure every use of `<AccountProfile>` is passing in the `profile` prop. I believe there are quite a few areas in our app that don't pass in the `profile` prop to the component and rely on this fallback logic to fetch it based on `accountId`.

Reply:
I think it's worth to use `Social.get(`${accountId}/profile/**`, "final")` here

Reply:
Thank you for the reviews, I reverted the changes for `Social.get`